js/models/adaptModel.js

```diff
@@ -468,6 +468,8 @@ export default class AdaptModel extends LockingModel {
 
     children.models.forEach(child => {
 
+      if (!child) return;
+
       if (!child.hasManagedChildren) {
```

Comment:
Think we should add the optional chaining operator here instead of adding the new if conditional statement, e.g.`if (!child?.hasManagedChildren) {`

Reply:
This would still push an `undefined` onto the descendants array (after the if)